packages/transport/src/core/types/PasswordProtectionCreationParameters.ts

```diff
@@ -1,8 +1,25 @@
 import { ISerializable, Serializable, serialize, validate } from "@js-soft/ts-serval";
 
+type Enumerate<N extends number, Acc extends number[] = []> = Acc["length"] extends N ? Acc[number] : Enumerate<N, [...Acc, Acc["length"]]>;
+type IntRange<From extends number, To extends number> = Exclude<Enumerate<To>, Enumerate<From>>;
+
+export enum PasswordLocationIndicatorOptions {
+    RecoveryKit = 0,
+    Self = 1,
+    Letter = 2,
+    RegistrationLetter = 3,
+    Email = 4,
+    SMS = 5,
+    App = 6,
+    Website = 7
+}
```

Comment:
this has to be moved away from here as it has nothing to do with the transport lib anymore.